build_route_info.sh

```diff
@@ -17,7 +17,8 @@ IFS=$'\n'
 for osm in uk_osm/out/*; do
 	geojson=$(basename $osm .osm).geojson
 	out=$(basename $osm .osm).bin
-	pueue add -w importer --escape cargo run --release -- "../$osm" "../uk_osm/$geojson" "../route_info_files/$out"
+	task=$(pueue add --print-task-id --working-directory importer --escape cargo run --release -- "../$osm" "../uk_osm/$geojson" "../route_info_files/$out")
+	pueue add --after $task --escape gzip "route_info_files/$out"
```

Comment:
It might be nicer to have just one task per area, and do `&& gzip foo`, but the sheer effort of figuring out how to escape all the shell stuff... https://github.com/Nukesor/pueue/wiki/Get-started#dependencies is easier